lib/FFI/CheckLib.pm

```diff
@@ -74,6 +74,12 @@ else
     \@DynaLoader::dl_library_path;
   };
   die $@ if $@;
+  if($os eq 'darwin' && (qx`command -v brew`)[0])
+  {
+    chomp(my $brew_path = (qx`brew --prefix`)[0]);
+    $brew_path .= '/lib';
+    push @$system_path, $brew_path unless any { $_ eq $brew_path } @$system_path;
```

Comment:
Just curious if you apply this patch which version of `ssl` will it find?

Reply:
It probably won't work since this formula is not symlinked by default:

```
brew info openssl
==> openssl@3: stable 3.0.5 (bottled) [keg-only]
Cryptography and SSL/TLS Toolkit
https://openssl.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/openssl@3.rb
License: Apache-2.0
==> Dependencies
Required: ca-certificates ✔
==> Caveats
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  $(brew --prefix)/etc/openssl@3/certs

and run
  $(brew --prefix)/opt/openssl@3/bin/c_rehash

openssl@3 is keg-only, which means it was not symlinked into /opt/homebrew,
because macOS provides LibreSSL.
```